diff --git a/node_modules/buffer-equal-constant-time/index.js b/node_modules/buffer-equal-constant-time/index.js
index 5462c1f..b20b32d 100644
--- a/node_modules/buffer-equal-constant-time/index.js
+++ b/node_modules/buffer-equal-constant-time/index.js
@@ -1,7 +1,7 @@
 /*jshint node:true */
 'use strict';
 var Buffer = require('buffer').Buffer; // browserify
-var SlowBuffer = require('buffer').SlowBuffer;
+var SlowBuffer = require('buffer').SlowBuffer || Buffer;
 
 module.exports = bufferEq;
 
